### PR TITLE
Skip labels in new command definitions

### LINKED ISSUE #2785: Linter warns about duplicate label when using #1 in macros
### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you still see this issue. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes

## Describe the bug [Required]

The linter gives unexpected warnings when using `#1` in macros multiple times
`Warning: Duplicate label #1`.

### To Reproduce

1. Write
```latex
\newcommand{\setLabelA}[1]{\label{#1}}
\newcommand{\setLabelB}[1]{\label{#1}}
```
2. Check with linter
3. See warning: `Warning: Duplicate label #1`., see screenshot

### Expected behavior

Warning shouldn't be raised when macro parameter is used in the label.

## Screenshots

grafik

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: Windows 10
 - VS Code version: 1.5.7
 - Extension version: 8.1.9.2
 - TeX distribution version: MiKTeX 4.2.1

### Are you using VSCodium?

No

### Are you using the Snap or Flatpack versions of VS Code?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

### PR DESCRIPTION
**Pull request by jlelong**
Close #2785

### CODE REVIEW

**Comment by jlelong** [comment]
@tamuratak Would it make sense to add a function `isNewCommand` to `latex-utensils` ?

**Comment by tamuratak** [comment]
No. We don't have to treat `newcommand` specially to parse LaTeX documents. 

To interpret LaTeX documents, we have to treat `newcommand` specially. However, interpreting LaTeX documents is out of the scope of `latex-utensils`.

**Comment by jlelong** [comment]
OK, I get your point.

File: src/utils/utils.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import * as vscode from 'vscode'
 import * as path from 'path'
 import * as fs from 'fs'
 
+import {latexParser} from 'latex-utensils'
```
Comment (tamuratak) [inline src/utils/utils.ts:5]: We had  better use a type-only import:

```ts
import type {latexParser} from 'latex-utensils'
```

**Review by tamuratak — CHANGES_REQUESTED** [review]

**Review by tamuratak — APPROVED** [review]

**Pull request merged**